tests/functional/cocoapods/test_resolvers_cocoapods.py

```diff
@@ -163,7 +163,7 @@ def provider(request):
 
 
 def _format_conflicts(exc):
-    return {r.name for r in exc.requirements}
+    return {r.name for (r, p) in exc.causes}
```

Comment:
```suggestion
    return {r.name for r, _ in exc.causes}
```

It’s more obvious the second argument is intentionally unused IMO.